task_geo/data_sources/covid/south_korea/south_korea_patients.py

```diff
@@ -0,0 +1,55 @@
+import io
+
+import pandas as pd
+import requests
+
+
+def south_korea_patients_connector(*args, **kwargs):
```

Comment:
I'm not sure why the signature of the functions is with *args, and **kwargs if you are only expecting one argument. 
Also, this argument is a url, does this data source work with any url? I think this connector should take no arguments. Can you please update the signature for both connector and data source?

Reply:
Done, takes no argument now and the url has been set